src/KeyValidatorTrait.php

```diff
@@ -23,6 +19,12 @@
      */
     protected function validateKey($key)
     {
+        $matched = preg_match('/^[A-Za-z0-9_.]{1,64}$/', $key);
+
+        if (1 !== $matched) {
+            throw new InvalidArgumentException('Can\'t validate the specified key');
```

Comment:
I think, we should not include origin key because we don't know there exception will be catched. It is probable that the message includes JS can get directly into the browser.